robot/description/rviz/caffeine.rviz

```diff
@@ -127,100 +131,468 @@ Visualization Manager:
+          Enabled: true
+          Name: Odometry
+        - Class: rviz/Group
+          Displays:
+            - Class: rviz/Camera
+              Enabled: true
+              Image Rendering: background and overlay
+              Image Topic: /zed/zed_node/left/image_rect_color
+              Name: Camera
+              Overlay Alpha: 0.5
+              Queue Size: 2
+              Transport Hint: raw
+              Unreliable: false
+              Value: true
+              Visibility:
+                Grid: false
+                Navigation:
+                  Current Goal: true
+                  Global Costmap: true
+                  Global Plan: true
+                  Local Costmap: true
+                  Local Plan: true
+                  Particle Cloud: true
+                  Planner Plan: true
+                  Robot Footprint: true
+                  Static Map: true
+                  Value: true
+                RobotModel: false
+                Sensors:
+                  Camera:
+                    Camera: true
+                    DepthImage: true
+                    StereoPointCloud: true
+                    Value: true
+                  Imu: true
+                  LaserScan: true
+                  Odometry:
+                    Encoders: true
+                    OdomFiltered: true
+                    Value: true
+                    VisualOdom: true
+                  Value: true
+                TF: false
+                Value: true
+              Zoom Factor: 1
+            - Class: rviz/Image
+              Enabled: false
+              Image Topic: /zed/zed_node/depth/depth_registered
+              Max Value: 25
+              Median window: 5
+              Min Value: 0
+              Name: DepthImage
+              Normalize Range: false
+              Queue Size: 2
+              Transport Hint: raw
+              Unreliable: false
+              Value: false
+            - Alpha: 1
+              Autocompute Intensity Bounds: true
+              Autocompute Value Bounds:
+                Max Value: 10
+                Min Value: -10
+                Value: true
+              Axis: Z
+              Channel Name: intensity
+              Class: rviz/PointCloud2
+              Color: 255; 255; 255
+              Color Transformer: RGB8
+              Decay Time: 0
+              Enabled: false
+              Invert Rainbow: false
+              Max Color: 255; 255; 255
+              Max Intensity: 4096
+              Min Color: 0; 0; 0
+              Min Intensity: 0
+              Name: StereoPointCloud
+              Position Transformer: XYZ
+              Queue Size: 10
+              Selectable: true
+              Size (Pixels): 3
+              Size (m): 0.009999999776482582
+              Style: Points
+              Topic: /zed/zed_node/point_cloud/cloud_registered
+              Unreliable: false
+              Use Fixed Frame: true
+              Use rainbow: true
+              Value: false
```

Comment:
Maybe leave as true so we don't have to click it? But is arguable

Reply:
So one thing about point clouds is that they are rather computationally expensive to render (ie. RViz gets a bit laggy), so I decided to keep it off by default, as seeing the point cloud isn't of utmost importance on every session. I would keep it on if it didn't have too much overhead, as I do like seeing it as well.

Though feel free to enable this on a separate branch if need be, just be mindful of not committing that change to master